ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/README.md

```diff
@@ -0,0 +1,88 @@
+<!--
+Copyright 2022 Adobe
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+Adaptive Form Accordion (v1)
+====
+Adaptive Form Accordion component written in HTL.
+
+## Features
+
+* Allows addition of accordion items of varying resource type.
+* Allowed components can be configured through policy configuration.
+* Toggle accordion panels from accordion header controls.
+* Ability to force a single panel to be displayed.
+* First Item is expanded by default.
+
+### Use Object
+The Accordion component uses the `com.adobe.cq.forms.core.components.models.form.Accordion` Sling model as its Use-object.
+
+### Edit Dialog Properties
+The following properties are written to JCR for this Accordion component and are expected to be available as `Resource` properties:
+
+1. `./jcr:title` - defines the label to use for this panel
+2. `./name` - defines the name of the panel, which will be submitted with the form data
+3. `./layout` - defines the layout type, either `simple` (default) or `responsiveGrid`; if no value is defined, the component will fallback to the value defined by the component's policy
+4. `./bindref` - defines the data binding, and how data will be sent
+5. `./visible` - defines initial state of panel visibility
+6. `./enabled` - defines initial state of panel if its enabled or not
+7. `./minItems` - defines minimum number of panel to be repeated
+8. `./maxItems` - defines maximum number of panel to be repeated
+9. `./tooltip` - defines tooltip on panel title
+10. `./description` - defines a help message that can be rendered in the field as a hint for the user
+
+## Client Libraries
+The component provides a `core.forms.components.accordian.v1` client library category that contains a recommended base
+CSS styling and JavaScript component. It should be added to a relevant site client library using the `embed` property.
+
+It also provides a `core.forms.components.accordian.v1.editor` editor client library category that includes JavaScript
+handling for dialog interaction. It is already included by its edit and policy dialogs.
+
+## BEM Description
+```
+BLOCK cmp-accordion
```

Comment:
what about Accordion title, long description, short description?

Reply:
added question mark. long and short description.
accordion title is already there.